### PR TITLE
Addition in troubleshooting docs

### PR DESCRIPTION
**Pull request by ayusharma**
I have added some errors in troubleshooting documentation.

### CODE REVIEW

File: docs/troubleshooting.rst [hunk 1]
```diff
@@ -32,3 +32,19 @@ because one if its dependencies is missing.
 To fix this, you can either install it locally or upgrade your version of pip::
 
   $ pip install --upgrade pip
+
+socket.error: [Errno 48] Address already in use
+===============================================
+Kinto runs on default port 8888 and this error is produced when another service has occupied port 8888
```
Comment (almet) [inline docs/troubleshooting.rst]: nit: missing the "." at the end of the sentence.

File: docs/troubleshooting.rst [hunk 2]
```diff
@@ -32,3 +32,19 @@ because one if its dependencies is missing.
...
+To fix this, see which service is running on port 8888::
+
+$ sudo lsof -i :8888
+
+and kill the process using PID from output::
+
+$ kill -kill [PID]
+
+cliquet.storage.exceptions.BackendError: OperationalError [Postgres Service]
+===========================================================================
```
Comment (almet) [inline docs/troubleshooting.rst]: you are missing a "=" here.

File: docs/troubleshooting.rst [hunk 3]
```diff
@@ -32,3 +32,19 @@ because one if its dependencies is missing.
...
+This error is produced when there is a problem to connect to postgres database or postgres service is not running.
```
Comment (almet) [inline docs/troubleshooting.rst]: nit: Add a blank line after the title

Comment (almet) [inline docs/troubleshooting.rst]: You could add a way to solve this: "Make sure sure the postgres server is running properly".

**Comment by almet** [comment]
Thanks a lot for this contribution! I've added a few comments here and there. In addition to them, please try to wrap all the lines to 80 chars.

**Comment by ayusharma** [comment]
Could I use the same branch to generate new pull request by hard resetting the old commits ?

**Comment by Natim** [comment]
Yes you can add new commit to your branch and it will automatically update this pull-request.

**Comment by leplatrem** [comment]
Excellent!

Thanks!

File: docs/troubleshooting.rst [hunk 4]
```diff
@@ -32,3 +33,23 @@ because one if its dependencies is missing.
...
+
+Another process has occupied Kinto's default port 8888.
+
+To fix this, see which service is running on port 8888::
+
+$ sudo lsof -i :8888
+
+and kill the process using PID from output::
+
+$ kill -kill [PID]
```
Comment (Natim) [inline docs/troubleshooting.rst]: You can also use: ``fuser -k 8888/tcp``

**Pull request merged**